dub.sdl

```diff
@@ -5,8 +5,8 @@ license "MIT"
 copyright "Copyright © 2016"
 targetType "sourceLibrary"
 
-dependency "dpq2" version="~>1.0.0-alpha.11"
-dependency "vibe-d:core" version="~>0.8.3"
+dependency "dpq2" version=">=1.0.5"
+dependency "vibe-d:core" version="~>0.8.4"
```

Comment:
Not agreed with this line: someone can use old vibe.d

Reply:
so ">= 0.8.3" then?

Reply:
As I know, 0.8.4 also belongs inside of  ~>0.8.3, and 1.0.5 inside of ~>1.0.0-alpha.11

Reply:
My proposal is: change dpq2 to ~>1.0.0 and not touch other versions.

Reply:
but my hope is to get at least 1.0.3 so I can use columnExists for Row structs

Reply:
vibe.d.db.postgresql package should get lates version of dpq2
also you can define any version of dpq2 in your dub config to override this version